pkgs/by-name/lu/lux-cli/package.nix

```diff
@@ -0,0 +1,116 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  libgit2,
+  gnupg,
+  libgpg-error,
+  gpgme,
+  darwin,
+  installShellFiles,
+  luajit,
+  makeWrapper,
+  symlinkJoin,
+  lua51Packages,
+  lua52Packages,
+  lua53Packages,
+  lua54Packages,
+  nix,
+  versionCheckHook,
+}:
+let
+  lux-lua-bundle = symlinkJoin {
+    name = "lux-lua-bundle";
+    paths = [
+      lua51Packages.lux-lua
+      lua52Packages.lux-lua
+      lua53Packages.lux-lua
+      lua54Packages.lux-lua
+    ];
+  };
+in
+rustPlatform.buildRustPackage rec {
+  pname = "lux-cli";
+
+  version = "0.3.13";
+
+  src = fetchFromGitHub {
+    owner = "nvim-neorocks";
+    repo = "lux";
+    rev = "v${version}";
+    hash = "sha256-G/j6Tz6mZwdpo7QmVF7Vw5BrEhTtKtzoLh0qkWXxjsQ=";
+  };
+
+  buildAndTestSubdir = "lux-cli";
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-duyaLqwiprrGATTA9ZmvzNaOhnrq1EdlCNobBUCFYUc=";
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgramArg = "--version";
+  doInstallCheck = true;
+
+  nativeBuildInputs = [
+    makeWrapper
+    pkg-config
+    installShellFiles
+  ];
+
+  buildInputs =
+    [
+      luajit
+      openssl
+      libgit2
+      gnupg
+      libgpg-error
+      gpgme
+    ]
+    ++ lib.optionals stdenv.isDarwin [
+      darwin.apple_sdk.frameworks.Security
+      darwin.apple_sdk.frameworks.SystemConfiguration
+    ];
```

Comment:
Are these not in the default darwin stdenv?

Reply:
In the crane build (run in the lux CI), the darwin builds fail without these.

Reply:
I can try playing around with it

Reply:
They may still be needed elsewhere if they happen to use an old nixpkgs checkout, but they are no longer needed since 24.11 so not sure why the (other) CI fails.

Reply:
I guess our crane build must have been pinned to an older nixpkgs when we first set it up.